common/app/model/content.scala

```diff
@@ -334,6 +334,7 @@ object Article {
     val javascriptConfig: Map[String, JsValue] = Map(
       ("contentType", JsString(contentType)),
       ("isLiveBlog", JsBoolean(content.tags.isLiveBlog)),
+      ("isMinuteArticle", JsBoolean(content.tags.isUSMinuteSeries)),
```

Comment:
Does every content item need this value set? This seems like a job for `conditionalConfig`